deepliif/models/networks.py

```diff
@@ -347,7 +347,7 @@ def __init__(self, input_nc, output_nc, ngf=64, norm_layer=nn.BatchNorm2d, use_d
         else:
             use_bias = norm_layer == nn.InstanceNorm2d
 
-        model = [nn.ReflectionPad2d(3), #nn.ReflectionPad2d(3),
+        model = [nn.ZeroPad2d(3) if os.environ['DEEPLIIF_SEED'] != 'None' else nn.ReflectionPad2d(3),
```

Comment:
This was done to make the model deterministic?  Would this affect reloading saved models now that you are changing the layout of the network?

Reply:
@drewmibm it will not, if you use the same setting when you run inference/reload (either put a seed or use no seed); but you are right, that means we need add this seed argument to the inference method as well

Reply:
@drewmibm I think we can submit it for now, and see whether the decision is to stick to deterministic training (so these padding layers won't vary)

Reply:
if not, we can then add this seed option to the inference method